sample/src/main/res/layout/activity_main.xml

```diff
@@ -22,6 +22,7 @@
         app:dayBackgroundColor="#05000000"
         app:todayBackgroundColor="#1848adff"
         app:headerColumnBackground="#ffffffff"
-        app:todayHeaderTextColor="@color/accent"/>
+        app:todayHeaderTextColor="@color/accent"
+        app:showHalfHours="false"/>
```

Comment:
You have implemented your own DateTimeInterpreter and interpretTime method correct? Then you need to implement it to properly show the minutes.